### PR TITLE
chore: Update extension docs with gifs for adding license and pragma version

### PR DESCRIPTION
**Pull request by antico5**

### CODE REVIEW

File: EXTENSION.md [hunk 1]
```diff
@@ -145,6 +145,14 @@ Two code actions will appear against a function with this error: _Add public vis
 
 ![Public Private](https://raw.githubusercontent.com/NomicFoundation/hardhat-vscode/main/docs/gifs/public-private.gif "Public Private")
 
+#### Adding license identifier and `pragma solidity` version
+
+When no license is specified on a contract, the `solidity(1878)` warning is raised by the compiler. Similarly, when no compiler version is specified with a `pragma solidity` statement, the compiler shows the `solidity()` warning. There are code actions available for quick fixes.
```
Comment (kanej) [inline EXTENSION.md:150]: compiler shows the `solidity(3420)` warning

Comment (antico5) [reply EXTENSION.md:150]: Nice catch! I was sleepy when writing this one :joy:

**Review by kanej — APPROVED** [review]
Looks good.

**Review by kanej — CHANGES_REQUESTED** [review]
Looks good. Can you add a link to this section via the toc at the top?

**Review by kanej — APPROVED** [review]

**Pull request merged**